td-paging/src/page_table.rs

```diff
@@ -108,7 +108,7 @@ pub fn create_mapping_with_flags(
             Size4KiB::SIZE
         };
 
-        sz -= mapped_size;
+        sz = sz.checked_sub(mapped_size).unwrap();
```

Comment:
This function returns a `Result<()>` so couldn't this be:

```suggestion
        sz = sz.checked_sub(mapped_size)?;
```
```suggestion
        sz = sz.checked_sub(mapped_size).unwrap();
```

Reply:
111 | |         sz = sz.checked_sub(mapped_size)?;
    | |                                         ^ cannot use the `?` operator in a function that returns `core::result::Result<(), Error>`

Reply:
hi @liuw1 , should we return error or unwrap here? Why not change `sz` as a signed integer. If we don't accept `sz` less than `0x1000`, we should check it at https://github.com/confidential-containers/td-shim/blob/main/td-paging/src/page_table.rs#L84

Reply:
Resolved